gui/setup.iss

```diff
@@ -39,27 +39,27 @@ Name: "desktopicon"; Description: "{cm:CreateDesktopIcon}"; GroupDescription: "{
 [Files]
 ; This isn't as wreckless as it seems; these directories are populated on a
 ; controlled way by the batch file.
-Source: qtdir\bin\*.dll;		DestDir: "{app}"; Flags: ignoreversion
-Source: qtdir\plugins\*; 		DestDir: "{app}\plugins"; Flags: ignoreversion recursesubdirs createallsubdirs
+;Source: qtdir\bin\*.dll;		DestDir: "{app}"; Flags: ignoreversion
+;Source: qtdir\plugins\*; 		DestDir: "{app}\plugins"; Flags: ignoreversion recursesubdirs createallsubdirs
 
 Source: gmapbase.html; 			DestDir: "{app}"; Flags: ignoreversion
 Source: qt.conf;       			DestDir: "{app}"; Flags: ignoreversion
 
-Source: ..\build-app-Desktop_Qt_5_2_1_Mingw_32bit-Release\release\gpsbabelfe.exe; 	DestDir: "{app}"; Flags: ignoreversion
-Source: ..\..\build-GPSBabel-Desktop_Qt_5_2_1_Mingw_32bit-Release\release\gpsbabel.exe;   	DestDir: "{app}"; Flags: ignoreversion
+Source: ..\build-app-Desktop_Qt_5_5_1_Mingw_32bit-Release\release\*; Excludes: "*.cpp,*.o"; DestDir: "{app}"; Flags: ignoreversion recursesubdirs createallsubdirs

```

Comment:
Doesn't this make it more easy to accidentally ship files we didn't mean to ship? Granted, in an automated environment, that's less of a big deal than on our workstations where we may have a stray script file or editor backup.

Reply:
I tried a few changes to hopefully mollify your concerns:
1) I explicitly delete the build directories we will be getting stuff out of before filling them with the make. This isn't necessary in the appveyor environment, at least without build caches, but I think it makes the intent more explicit.
2) I have inno setup create a manifest and make it an artifact.  It can be examined to see what got included in the setup executable.

Reply:
I like it.